lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -372,6 +385,24 @@ export default {
 	},
 	computed: {
 		iconSize(): string {
+			if (this.state === ICON_BUTTON_STATES.LOADING) {
```

Comment:
It need have separated condition with return at the end.

Reply:
are you sure these sizes are different from `ICON_ONLY_ICON_SIZES_MAP`?

Reply:
BTW. Only `ICON_ONLY` type should have these sizes

Reply:
> BTW. Only ICON_ONLY type should have these sizes, see:

@kvas-damian are You sure? `Outlined` and `Filled` are using same icon and have same sizes (5 types). This loader should looks same for `iconOnly` and other variants.

Reply:
> are you sure these sizes are different from ICON_ONLY_ICON_SIZES_MAP?

Right! Condition should be:

```
			if (this.type === ICON_BUTTON_TYPES.ICON_ONLY || this.state === ICON_BUTTON_STATES.LOADING) {
				return ICON_ONLY_ICON_SIZES_MAP[this.size];
			}
```

Reply:
For `filled L` I see Icon XS: https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=4803%3A83076

Reply:
Fixed.